pkg/server/sqlflowserver.go

```diff
@@ -168,7 +168,7 @@ func parse2Any(val interface{}) (proto.Message, error) {
 		return &wrappers.BytesValue{Value: v}, nil
 	case time.Time:
 		return &pyts.Timestamp{
-			Seconds: int64(v.Second()),
+			Seconds: int64(v.Unix()),
```

Comment:
Thank you for pointing this out!

I tried to understand the change by reading https://godoc.org/github.com/golang/protobuf/ptypes/timestamp#Timestamp.GetSeconds and saw the example code:

```
Timestamp timestamp;
timestamp.set_seconds(time(NULL));
timestamp.set_nanos(0);
```

Where the POSIX call `time` returns the UNIX seconds.